geaflow/geaflow-dsl/geaflow-dsl-connector/geaflow-dsl-connector-hbase/src/main/java/com/antgroup/geaflow/dsl/connector/hbase/HBaseConstants.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2023 AntGroup CO., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.antgroup.geaflow.dsl.connector.hbase;
+
+public class HBaseConstants {
+
+    public static final int DEFAULT_BUFFER_SIZE = 1024 * 1024;
+
+    public static final String DEFAULT_NAMESPACE = "default";
+
+    public static final String DEFAULT_COLUMN_FAMILY = "GeaFlow";

```

Comment:
Why we should have default column family 'geaflow' ? It should be specified by the user

Reply:
If the user does not specify the corresponding column family for a column in `geaflow.dsl.hbase.family_name`, the default column family `geaflow` is used